packages/vite/src/node/cli.ts

```diff
@@ -130,6 +130,7 @@ cli
     `[boolean] force empty outDir when it's outside of root`
   )
   .option('-m, --mode <mode>', `[string] set env mode`)
+  .option('--watch', `[boolean] rebuilds when modules have changed on disk`)
```

Comment:
**suggestion (if-minor):** Webpack also provides a `-w` shorthand
We may also provide this